demosplan/DemosPlanCoreBundle/Logic/DefaultTwigVariablesService.php

```diff
@@ -215,6 +215,7 @@ public function loadVariables(Request $request): void
             'urlPathPrefix'                    => $this->globalConfig->getUrlPathPrefix(),
             'urlScheme'                        => $this->globalConfig->getUrlScheme() ?? $request->getScheme(),
             'useOpenGeoDb'                     => $this->globalConfig->getUseOpenGeoDb(),
+            'main_page_external_links'         => $this->globalConfig->getMainPageExternalLinks(),
```

Comment:
Please stick to the naming scheme.

Reply:
done in 0f36bf8d50b0b713dab6ca1a4d26bff6531af83a